BLUESPAWN-client/src/user/BLUESPAWN.cpp

```diff
@@ -173,12 +174,40 @@ void print_help(cxxopts::ParseResult result, cxxopts::Options options) {
 	}
 }
 
+void Bluespawn::check_correct_arch() {
+	BOOL bIsWow64 = FALSE;
+	if (IsWindows10OrGreater()) {
+		typedef BOOL (WINAPI *LPFNIsWow64Process2) (HANDLE, PUSHORT, PUSHORT);
+
+		LPFNIsWow64Process2 fnIsWow64Process2;
+		USHORT ProcessMachine;
+		USHORT NativeMachine;
+
+		fnIsWow64Process2 = (LPFNIsWow64Process2)GetProcAddress(GetModuleHandle(TEXT("kernel32")), "IsWow64Process2");
```

Comment:
reinterpret_cast.

Also, see the linker in commonlib

Reply:
fixed by using DynamicLinker I think